### ISSUE TITLE
clusterbats: nodeset before nodes get discovered

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#595]
**Version:** r7_login_image 9c77728
**Environment:** tmaster

```
not ok 14 1.2.5 We can assign the containers to the default virtual cluster a
# (in test file /trinity/testing/clusterbats/t1.2.bats, line 74)
#   `sshpass -p 'system' ssh -o StrictHostKeyChecking=no login.vc-a systemctl restart slurm' failed with status 255
# Warning: Unable to find mac address for node001
# Warning: Unable to find mac address for node002
# Warning: Unable to find mac address for node001
# Warning: Unable to find mac address for node002
# node001: netboot centos7-x86_64-trinity
# node002: netboot centos7-x86_64-trinity
```

inspecting the nodes, they are in genesis: so all subsequent tests will fail

```
[root@controller(tmaster) ~]# ssh node001
Warning: Permanently added 'node001,10.141.0.1' (RSA) to the list of known hosts.
[xCAT Genesis running on node001 /]# exit
```

**Comment by hansthen** [comment]
I think this is fixed in  b1a6a46

**Comment by themisa** [comment]
Fixed. tested in:
[root@controller(tmaster) ~]# /trinity/report
**Version:** r7 b55e0e1
**Environment:** tmaster

```
node001: ok 18 1.2.0.a We can configure the switch
node001: ok 19 1.2.1 We can discover compute nodes
node001: ok 20 1.2.5 We can assign the containers to the default virtual cluster a
node001: ok 21 1.2.6 There is a virtual login node
node001: ok 22 1.2.7 Slurm and munge are running on the virtual login nodes
node001: ok 23 1.2.8 The compute nodes can connect to the internet
```
